evergreen.yml

```diff
@@ -238,6 +237,16 @@ tasks:
   - name: validate_genny_tasks
     commands:
       - func: f_fetch_source
+      - command: github.generate_token
+        params:
+          owner: 10gen
+          repo: dsi
+          expansion_name: dsi_x_access_token
+      - command: github.generate_token
+        params:
+          owner: 10gen
+          repo: PrivateWorkloads
+          expansion_name: private_workloads_x_access_token
```

Comment:
We currently use the same github app we use in SPS here. I've requested a new app for this purpose and will update the evergreen config accordingly once it is available.